test_scripts/API/SetGlobalProperties/026_ATF_UI_IsReady_no_resp_SetGlobalProp_UI_SetGlobalProp_UNSUP_RES.lua

```diff
@@ -0,0 +1,161 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one component of RPC
+-- [UI Interface] HMI does NOT respond to IsReady and mobile app sends RPC that must be splitted
+--
+-- Description:
+-- test is intended to check that SDL sends UNSUPPORTED_RESOURCE (success:true) to mobile app in case HMI respond: WARNINGS one HMI-portion and UNSUPPORTED_RESOURCE to another one
+-- in particular test it is checked case when TTS.SetGlobalProperties with WARNINGS and to UI.SetGlobalProperties with UNSUPPORTED_RESOURCE (success:true)
+--
+-- 1. Used preconditions:
+-- HMI does not respont to UI.IaReady
```

Comment:
@istoimenova , script was updated in commit 9e46219c5c10e19aa72a3cb92276079989e650fe